pkg/TIPA.py

```
@@ -11,33 +11,31 @@
-import sys
-import time
-import string
-import struct
 import re
-import requests
-import pytesseract
-import cv2
 import tempfile
+import threading
+import time
 import urllib
```

Ruff (0.8.0)

`urllib` imported but unused

Remove unused import: `urllib`

(F401)